releng/org.eclipse.xtext.target/org.eclipse.xtext.target-oxygen.target

```diff
@@ -32,7 +32,7 @@
 		<unit id="org.eclipse.xtext.testlanguages.ide.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.tests" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.tests.source" version="0.0.0"/>
-		<repository location="https://ci.eclipse.org/xtext/job/xtext-core/job/master/lastStableBuild/artifact/build/p2-repository/"/>
```

Comment:
is useless as domain model build does not run on build

Reply:
Thanks for the hint, I reverted that change.